mongo_startup.sh

```diff
@@ -490,42 +503,47 @@ start_replicaset(){
   local RSBASEPORT="$3"
   local EXTRA="$4"
   mkdir -p "${RSDIR}"
-  if [ ${RS_HIDDEN} = 1 ] || [ ${RS_DELAYED} = 1 ]; then
+  if [ ${RS_HIDDEN} = 1 ] || [ ${RS_DELAYED} = 1 ] || [ ${RS_ARBITER} = 1 ]; then
     nodes_count=(1 2 3 4)
   else
     nodes_count=(1 2 3)
   fi
   echo -e "\n=== Starting replica set: ${RSNAME} ==="
-#  for i in 1 2 3; do
   for i in "${!nodes_count[@]}"; do
```

Comment:
with this the node names are changed from `node1`, `node2`, `node3` to `node0`, `node1`, `node2` - since now we would use array index instead of value in node naming, not sure if this was intentional or not.